app/views/student_identifier_types/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render 'organizations/organization_banner' %>
+
+<div class="site-content">
+  <div class="site-content-cap">
+    <h2 class="site-content-heading">
+      New identifier type
+    </h2>
+  </div>
+</div>
+
+<div class="site-content-body">
+  <div class="flash flash-with-icon flash-form">
+    <%= octicon 'alert' %>
+    Please do not ask students to provide any confidential information (e.g. SSN or bank account number) as their identifiers.
+  </div>
+  <%= form_for [@organization, @student_identifier_type] do |f| %>
+      <%= render partial: 'student_identifier_types/form_options', locals: { f: f } %>
+      <div class="form-actions">
+        <%= f.submit 'Create Identifier', class: 'btn btn-green js-navigation' %>
+        <%= link_to 'Cancel', session[:return_to], class: 'btn', role: 'button'%>
```

Comment:
@tarebyte Yep. A teacher can click into the page from the student identifier type management page or from the assignment creating page:

![](https://cloud.githubusercontent.com/assets/1311594/16685545/691876ec-453e-11e6-9ad3-98b24dcee0a9.png)

![screen shot 2016-07-12 at 10 07 40 am](https://cloud.githubusercontent.com/assets/1311594/16753052/0249e288-4819-11e6-9c4f-4ee54a90457c.png)